tests/FSharp.Compiler.ComponentTests/Signatures/TestCasesForGenerationRoundTrip/access.fsx

```diff
@@ -0,0 +1,287 @@
+                {
+                    Datum: int
+                }
+
+        type public Datum = 
+            internal
+                {
+                    Thing: Data
+                }
+
+        type public Datum2 = 
+            internal | A of Data * Data | B of Data
+                
+    module public Test2 =
+        
+        type internal Data = 
+                {
+                    Datum: int
+                }
+
+        type internal Datum = 
+                {
+                    Thing: Data
+                }
+
+        type internal Datum2 = 
+             | A of Data * Data | B of Data
+                
+    module public Test3 =
+        
+        type public Data = 
+              internal
+                {
+                    Datum: int
+                }
+
+        type internal Datum = 
+                {
+                    Thing: Data
+                }
+
+        type internal Datum2 = 
+            internal | A of Data * Data | B of Data
+                
+
+    module public Test4 =
+        
+        type internal Data = 
+                {
+                    Datum: int
+                }
+
+        type public Datum = 
+             internal
+                {
+                    Thing: Data
+                }
+
+        type public Datum2 = 
+            internal | A of Data * Data | B of Data
+                
+
+    module public Test5 =
+        
+        type private Data = 
+                {
+                    Datum: int
+                }
+
+        type public Datum = 
+            private
+                {
+                    Thing: Data
+                }
+
+        type public Datum2 = 
+            private | A of Data * Data | B of Data
+
+
+    module Test6 = 
+        module internal HelperModule = 
+            
+            type public Data = 
+                private
+                    {
+                        Datum: int
+                    }
+                    
+            let internal handle (data:Data): int = data.Datum
+            
+        module public Module =
+            
+            type public Data = 
+                private
+                    {
+                        Thing: HelperModule.Data
+                    }
+                    
+            let public getInt (data:Data): int = HelperModule.handle data.Thing               
+
+    module Test7 = 
+        module internal HelperModule = 
+            
+            type Data = 
+                    {
+                        Datum: int
+                    }
+                    
+            let handle (data:Data): int = data.Datum
+            
+        module Module =
+            
+            type Data = 
+                internal
+                    {
+                        Thing: HelperModule.Data
+                    }
+                    
+            let getInt (data:Data): int = HelperModule.handle data.Thing               
+
+
+    (*--------------------*)  
+
+#if TESTS_AS_APP
```

Comment:
Just double-checking if the constants are still applicable / still apply after all the file migration?

Reply:
Was just a file move, with the **intention to move as many things as is**, without a change.

Reply:
Well I just don't know how and where this constant is defined. Sometimes constants are defined in project files, so when files are moved from one project to another project, some constants should be copied. But I don't know if that's the case here, hence double-checking :)

Reply:
The code driving the suite is where all the important stuff happens.
Defines are part of it:

```
    Fsi generatedSignature    
    |> withAdditionalSourceFile (FsSource implContents)
    |> withLangVersionPreview
    |> withDefines ["TESTS_AS_APP";"COMPILED"]
```

Reply:
Ah okay I see now, thanks, it's fine then